### PR TITLE
Add French comment

### PR DESCRIPTION
**Pull request by felinben**

### CODE REVIEW

**Comment by WeedyWeedSmoker** [comment]
Ok pour la traduction ! J'aurais mis un accent à multimédia par contre… Merci !

File: retroarch.desktop [hunk 1]
```diff
@@ -5,6 +5,7 @@ GenericName=RetroArch
 Type=Application
 Comment=Frontend for emulators, game engines and media players
 Comment[ru]=Графический интерфейс для эмуляторов, игровых движков и медиаплееров
+Comment[fr]=Interface graphique pour émulateurs, moteurs de jeu et lecteurs multimedia
```
Comment (RobLoach) [inline retroarch.desktop]:
``` suggestion
Comment[fr]=Interface graphique pour émulateurs, moteurs de jeu et lecteurs multimédia
```

**Comment by fpscan** [comment]
@RobLoach is this what you see when you right click the icon and get in specifications?

**Comment by RobLoach** [comment]
@fpscan Yeah... You can see the spec here:
https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s05.html

**Pull request merged**